src/OrchardCore.Modules/OrchardCore.Autoroute/Startup.cs

```diff
@@ -133,3 +134,12 @@ public override void ConfigureServices(IServiceCollection services)
         services.AddShapeTableProvider<WidgetAutorouteShapeTableProvider>();
     }
 }
+
+[RequireFeatures("OrchardCore.Autoroute")]
+public class AutoPartStartup : StartupBase
```

Comment:
I suppose this should rather be?

```suggestion
[RequireFeatures("OrchardCore.ContentLocalization")]
public class LocalizationStartup : StartupBase
```

Reply:
`ContentLocalizationStartup` will be better, to avoid the conflict between localization & content localization modules

Reply:
BTW I didn't notice there was other feedback except your latest comments. Are there any?

Reply:
> `ContentLocalizationStartup` will be better, to avoid the conflict between localization & content localization modules

Sure.


> BTW I didn't notice there was other feedback except your latest comments. Are there any?

No.